libraries/Microsoft.Bot.Connector/Authentication/GovernmentAuthenticationConstants.cs

```diff
@@ -16,7 +16,7 @@ public static class GovernmentAuthenticationConstants
         /// <summary>
         /// TO GOVERNMENT CHANNEL FROM BOT: Login URL.
         /// </summary>
-        public const string ToChannelFromBotLoginUrl = "https://login.microsoftonline.us/cab8a31a-1906-4287-a0d8-4eef66b95f6e";
+        public const string ToChannelFromBotLoginUrl = "https://login.microsoftonline.us/MicrosoftServices.onmicrosoft.us";
```

Comment:
does casing matter here? URI Segments are normally not case sensitive, but I'm worried the casing here might cause comparison issues.